USER_GUIDE.md

```diff
@@ -108,6 +109,90 @@ client
     .await?;
 ```
 
+### Make Raw Json Requests
+
+To invoke an API that is not supported by the client, use `client.send` method to do so. see [examples/json](./opensearch/examples/json.rs) for a complete working example
+
+#### PUT
+```rust
+// create index
+let response = client
+    .send(
+        Method::Put,
+        "/movies",
+        HeaderMap::new(),
+        Option::<&String>::None,
+        Option::<&String>::None,
+        None,
+    )
+    .await?;
+```
+
+#### POST
+
+```rust
+// Add a document
+let document =
+    r#"{
+      "name" : "Mission Impossible",
+      "year" : "2001"
+    }"#;
+
+let response = client
+    .send(
+        Method::Post,
+        "/movies/_doc",
+        HeaderMap::new(),
+        Option::<&String>::None,
+        Some(document),
+        None,
+    )
+    .await?;
+
+assert_eq!(response.status_code().as_u16(), 201_u16);
```

Comment:
Do we really have to cast these in Rust, can't write `assert_eq!(response.status_code(), 201)`?